test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -853,48 +849,48 @@ func runWorkload(ctx context.Context, tb testing.TB, tc *testCase, w *workload,
 
 	if cleanup {
 		// This must run before controllers get shut down.
-		defer cleanupWorkload(ctx, tb, tc, client, numPodsScheduledPerNamespace)
+		defer cleanupWorkload(tCtx, tc, numPodsScheduledPerNamespace)
 	}
 
-	for opIndex, op := range unrollWorkloadTemplate(tb, tc.WorkloadTemplate, w) {
+	for opIndex, op := range unrollWorkloadTemplate(tCtx, tc.WorkloadTemplate, w) {
 		realOp, err := op.realOp.patchParams(w)
 		if err != nil {
-			tb.Fatalf("op %d: %v", opIndex, err)
+			tCtx.Fatalf("op %d: %v", opIndex, err)
 		}
 		select {
-		case <-ctx.Done():
-			tb.Fatalf("op %d: %v", opIndex, ctx.Err())
+		case <-tCtx.Done():
+			tCtx.Fatalf("op %d: %v", opIndex, context.Cause(tCtx))
```

Comment:
Q: why not implement the same method?

Reply:
The same method as what? Sorry, I don't follow.